components/dada-lex/src/lines.rs

```diff
@@ -36,6 +36,7 @@ pub fn line_column(db: &dyn crate::Db, filename: Filename, position: Offset) ->
 }
 
 #[salsa::memoized(in crate::Jar ref)]
+#[allow(clippy::needless_lifetimes)]
```

Comment:
...this is annoying, maybe I will tweak the salsa output to generate that lint, or to avoid it

Reply:
Yes I just dropped those `#[allow(...)]`s here to make Clippy happy. Fixing it in salsa is the way to go.